students/BrianNisonger/Lesson7/assignment/parallel.py

```diff
@@ -0,0 +1,55 @@
+from multiprocessing.pool import ThreadPool
+from database import import_data
+from database import show_available_products
+
+
+def run_parallel():
+    """
+    Run the import data in parallel
+    """
+    pool = ThreadPool()
```

Comment:
Aha! I think the problem is you have to initiate it with how many threads you want. The default is probably one. Try:
```
pool = ThreadPool(3)
```
and see if that speeds things up.